docs/shared/recipes/running-tasks/executors-and-configurations.md

```diff
@@ -1,18 +1,10 @@
-# Use Task Executors
+# Executors and Configurations
```

Comment:
Why did we move this to a recipe? It is still kind of a concept page explaining at a higher level what an executor is and various aspects of it (config, running etc..).